pkgs/top-level/all-packages.nix

```
@@ -5973,6 +5973,10 @@ with pkgs;
 
   html-proofer = callPackage ../tools/misc/html-proofer { };
 
+  htmlq = callPackage ../development/tools/htmlq {
+    inherit (pkgs.darwin.apple_sdk.frameworks) Security;
```

```suggestion
    inherit (darwin.apple_sdk.frameworks) Security;
```

I think `pkgs` is not required.

---

Oh I just saw this comment. Looks like there are several occurrences of this in `all-packages.nix` so a file-wide change might be in order.

---

Ah, doesn't look like there is much of a convention.  

>Looks like there are several occurrences of this in all-packages.nix so a file-wide change might be in order.

Good call.